apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx

```diff
@@ -331,6 +332,21 @@ export const DeviceActionDefaultRendering = <R, H extends States, P>({
     }
   }
 
+  if (
+    device?.modelId === "nanoS" &&
+    (
+      request as {
```

Comment:
you can create a nanoS based request type and use it in this casting rather than inline cast/type